src/github/client/github-client.ts

```diff
@@ -98,13 +98,16 @@ export class GitHubClient {
 		}
 	}
 
-	protected async graphql<T>(query: string, config: AxiosRequestConfig, variables?: Record<string, string | number | undefined>): Promise<AxiosResponse<GraphQlQueryResponse<T>>> {
+	protected async graphql<T>(query: string, config: AxiosRequestConfig, variables?: Record<string, string | number | undefined>, metrics?: Record<string, string>): Promise<AxiosResponse<GraphQlQueryResponse<T>>> {
 		const response = await this.axios.post<GraphQlQueryResponse<T>>(this.graphqlUrl,
 			{
 				query,
 				variables
 			},
-			config);
+			Object.assign({}, {
```

Comment:
Curious what's Object.assign for?

Reply:
It is exactly like bellow. But ts wont' complain.
```
{
 ... config,
metrics:
}
```

Reply:
Eventually both will endup with a new object.